tests/acceptance/repo/trigger-build-test.js

```diff
@@ -104,10 +104,6 @@ module('Acceptance | repo/trigger build', function (hooks) {
 
     assert.ok(triggerBuildPage.popupIsVisible, 'modal is visible after click');
 
-    assert.equal(triggerBuildPage.branches.length, 1, 'expected the not-on-GitHub branch to be hidden');
-    assert.equal(triggerBuildPage.branches[0].value, 'master');
-
-    await triggerBuildPage.selectBranch('master');
```

Comment:
Yeah, it's expected, I removed it because the branch is now pre-selected, so it doesn't make sense to additionally select it in the test. But thanks for catching it! :)

Reply:
Cool, that makes sense. I thought it might be something like that :+1: